microsoft-edge/webview2/concepts/working-with-local-content.md

```diff
@@ -126,12 +173,45 @@ webView->Navigate(
 
 
 <!-- ====================================================================== -->
-## Navigate to an HTML string
+## Loading local content by navigating to an HTML string
 
 Another method to load local content is the `NavigateToString` method.  This approach loads the content into WebView2 directly from a string.  This can be useful if you will be packaging the content via the app code, or if you'd like to dynamically create the content.
 
 Another scenario where navigating to a string might be useful is if you want to load content that is not accessible via a URL. For example, if you have an in-memory representation of an HTML document, you could use the `NavigateToString` method to load that content into the WebView2 control.  This can be useful if you want to avoid the need to write the content to a file or server before loading it into the control.
 
+
+<!-- ------------------------------ -->
+#### Considerations for loading local content by navigating to an HTML string
+
+
+###### Origin-based DOM APIs
+
+`NavigateToString` has as its document URI `about:blank` and `null` as its origin. This means you cannot use any DOM API that depends on the origin, such as `localStorage` or `indexedDB`.
+
+
+###### DOM APIs requiring secure context
+
+Some newer browser features are limited to https URLs and are not available to the `about:blank` URI that `NavigateToString` uses. This includes webcam APIs, geolocation APIs, and notification APIs, among others. See [Secure contexts](https://developer.mozilla.org/en-US/docs/Web/Security/Secure_Contexts) for more information.
+
+
+###### Dynamic content
+
+When loading local content via `NavigateToString`, you are directly providing the local content as the method parameter. This means you are totally in control of the local content at runtime, and you can dynamically produce whatever local content you like.
+
+
+###### Additional web resources
+
+`NavigateToString` has the drawback that you can only specify the string content of the HTML document; there is no way to represent additional web resources, such as CSS, images, or script. Instead, to reference additional web resources from your HTML document, you must use one of the other mechanisms described in this article, or represent those additional web resources inline in the HTML document.
```

Comment:
```suggestion
`NavigateToString` has the drawback that you can only specify the string content of the HTML document; there's no way to represent additional web resources, such as CSS, images, or script. Instead, to reference additional web resources from your HTML document, you must use one of the other mechanisms described in this article, or represent those additional web resources inline in the HTML document.
```
corp style favors contraction unless need separate words for emphasis.  contraction seems to give suitable tone here